trackma/engine.py

```diff
@@ -314,7 +314,7 @@ def start(self):
                     # for later calls.
                     try:
                         self.msg.debug("Importing hook {}...".format(name))
-                        module = loader.find_module(name).load_module(name)
+                        module = loader.find_spec(name).loader.load_module(name)
```

Comment:
`load_module(fullname)` is deprecated and `exec_module(module)` is preferred. Can you change (and test) this code to use `exec_module` in addition to `importlib.util.module_from_spec` to create the module instance? As far as I understand, this how it is supposed to be used. See also #746.